### PR TITLE
Chapter1/homework/4

### PR DESCRIPTION
**Pull request by illayarajah**
Homework 4

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+                       
+            string firstName = returnString("Enter First Name");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Second person missing

**Review by Almantask — CHANGES_REQUESTED** [review]
Refer to other remarks from previous PR.

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (illayarajah) [reply Src/BootCamp.Chapter/Lesson3.cs]: I actually thought that we just do it twice to make sure the information is correct. 
i will add the 2nd person in.

**Comment by Almantask** [comment]
Looks good

**Pull request closed without merging**